### PR TITLE
ngx bootstrap cleanup

### PR DESCRIPTION
**Pull request by ezequielc**
## Description

- Cleanup prototype code
- Fix ngx-boostrap modal style
- Migrate streamline-icon to Angular
- Use streamline-icon in Angular

## Motivation and Context
Angular migration

## How Has This Been Tested?
Locally and on [stage-13]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by ezequielc** [comment]
@alex-deaconu I left a modal in Presentations List so you can review how it looks like, since it is not used in AngularJS yet. I will remove it in my next PR. Thanks

File: angular.json [hunk 1]
```diff
@@ -498,7 +498,6 @@
               "src/scripts/template-editor/directives/dtv-basic-uploader.js",
               "src/scripts/template-editor/directives/dtv-file-entry.js",
               "src/scripts/template-editor/directives/dtv-empty-file-list.js",
-              "src/scripts/template-editor/directives/dtv-streamline-icon.js",
```
Comment (alex-deaconu) [inline angular.json:501]: @ezequielc would it make sense to migrate the `streamline-icon` in the master branch directly, rather than this branch? Thanks.

**Review by alex-deaconu — APPROVED** [review]
@ezequielc looks good 👍 . One suggestion re: release order inline. Thanks.

File: angular.json (hunk 1 above)
Comment (ezequielc) [reply angular.json:501]: @alex-deaconu Yeah, good point, I should have done that. I guess at this point I will leave as is, given the extra effort. This branch should be in master around tomorrow. Thanks!

**Pull request merged**